partitioned-table.md

```diff
@@ -1350,7 +1455,9 @@ YEARWEEK()
 
 ### Compatibility with MySQL
 
-Currently, TiDB supports Range partitioning, Range COLUMNS partitioning, List partitioning, List COLUMNS partitioning, and Hash partitioning. Other partitioning types that are available in MySQL such as key partitioning are not supported yet in TiDB.
+Currently, TiDB supports Range partitioning, Range COLUMNS partitioning, List partitioning, List COLUMNS partitioning, Hash partitioning, and Key partitioning. Other partitioning types that are available in MySQL are not supported yet in TiDB.
+
+Currently, TiDB only supports single-column partitioning for Range COLUMNS partitioning and does not support using an empty partition column list for Key partitioning.
```

Comment:
```suggestion
Currently, TiDB does not support using an empty partition column list for Key partitioning.
```

Multi-column RANGE COLUMNS partitioning is supported, see #range-columns-partitioning